client/components/web-preview/content.jsx

```diff
@@ -406,21 +407,40 @@ export default class WebPreviewContent extends Component {
 					) }
 					{ 'seo' !== this.state.device && (
 						<div
+							onMouseEnter={ () => this.setState( { showIFrameOverlay: true } ) }
+							onMouseLeave={ () => this.setState( { showIFrameOverlay: false } ) }
 							className={ classNames( 'web-preview__frame-wrapper', {
 								'is-resizable': ! this.props.isModalWindow,
 							} ) }
 						>
+							{ /* eslint-disable jsx-a11y/mouse-events-have-key-events */ }
 							<iframe
 								ref={ this.setIframeInstance }
 								className="web-preview__frame"
-								style={ { ...this.state.iframeStyle, height: this.state.viewport?.height } }
+								style={ {
+									...this.state.iframeStyle,
+									height: this.state.viewport?.height,
+									pointerEvents: 'none',
```

Comment:
Setting `pointerEvents` to `none` might be a pretty big change. Off the top of my head, I can't see why this would be a problem, but do we need to cross check some other implementations of the web preview to make sure nothing is relying on this functionality?

Reply:
That's a good point, maybe we can set the pointer-events conditionally only for Launchpad just to be safe?